maven-surefire-plugin/src/site/markdown/java9.md

```diff
@@ -17,15 +17,6 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-Java 9 in JAVA_HOME
```

Comment:
You should bring back the history. Some users still use 2.20.1 and there was another behavior. We should add a new point for 2.21 and recommendations. Please add the list of Java API from Jira that the Oracle guy listed Maven artifacts, implementations. This is missing in Java documentation and people will needed to be announced about them.

Reply:
will do